src/Abp.AspNetCore/AspNetCore/Mvc/Validation/MvcActionInvocationValidator.cs

```diff
@@ -24,16 +25,27 @@ public void Initialize(ActionExecutingContext actionContext)
         {
             ActionContext = actionContext;
 
-            SetDataAnnotationAttributeErrors();
```

Comment:
This line forced to always add validation errors but you don't want in case you add a type to IgnoredTypes list. Was it really necessary?